coverage-tests/configuration/eg_configuration.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  name: 'eyes_selenium_java',
+  emitter: 'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/master/java/emitter.js',
+  overrides: [
+    'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/master/java/overrides.js',
+    'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/master/eg.overrides.js'
+  ],
+  template: 'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/master/java/template.hbs',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/eyes.sdk.java3/pullRequest?prid=7879168)
```suggestion
  template: "https://raw.githubusercontent.com/applitools/sdk.coverage.tests/master/java/template.hbs",
```